pkg/cmd/gist/view/view_test.go

```diff
@@ -47,6 +47,16 @@ func TestNewCmdView(t *testing.T) {
 				Filename: "cool.txt",
 			},
 		},
+		{
+			name: "files passed",
+			cli:  "-l 123",
+			tty:  true,
+			wants: ViewOptions{
+				Raw:      false,
+				Selector: "123",
+				Files:    true,
```

Comment:
This looks good. Could other cases in this test (the ones that existed before your PR) be expanded to assert that `Files` was false?